homeassistant/components/brother/strings.json

```
@@ -25,5 +25,111 @@
       "unsupported_model": "This printer model is not supported.",
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "status": {
+        "name": "Status"
+      },
+      "page_counter": {
+        "name": "Page counter"
+      },
+      "bw_page_counter": {
+        "name": "B/W page counter"
```

Wouldn't this be a more natural description?

```suggestion
        "name": "B/W pages"
```

---

Yeah, good idea.